src/System.Windows.Forms/src/System/Windows/Forms/PropertyGridInternal/CategoryGridEntry.CategoryGridEntryAccessibleObject.cs

```diff
@@ -31,8 +29,13 @@ public CategoryGridEntryAccessibleObject(CategoryGridEntry owningCategoryGridEnt
             /// </summary>
             /// <param name="direction">Indicates the direction in which to navigate.</param>
             /// <returns>Returns the element in the specified direction.</returns>
-            internal override UiaCore.IRawElementProviderFragment FragmentNavigate(UiaCore.NavigateDirection direction)
+            internal override UiaCore.IRawElementProviderFragment? FragmentNavigate(UiaCore.NavigateDirection direction)
             {
+                if (Parent is null)
+                {
+                    return null;
+                }
+
                 PropertyGridView.PropertyGridViewAccessibleObject parent = (PropertyGridView.PropertyGridViewAccessibleObject)Parent;
```

Comment:
A little cleaner would be write:
```suggestion
                if (Parent is not PropertyGridView.PropertyGridViewAccessibleObject parent)
                    return null;
```